README.md

```diff
@@ -162,6 +162,25 @@ docker exec palworld-server backup
 
 This will create a backup at `/palworld/backups/`
 
+## Setting Up Automatic Backups with Cron
+
+This section guides you through the process of configuring automatic backups using Cron. Use the provided command to set up the necessary Cron job for your Palworld server within a Docker environment.
+
+The configured backup system automatically deletes backups older than 7 days to manage storage efficiently.
+
+```bash
+docker exec palworld-server cron /etc/cron.d/backups-cron
```

Comment:
This requires manually running this command through exec, I would prefer to have a enviroment variable to enable/disable the cron backup funcionality:
ENABLE_BACKUP=true/false
BACKUP_INTERVAL="0 0 * * *"

Reply:
Ajusted